test/functional/apps/discover/classic/_doc_table.ts

```diff
@@ -194,16 +194,18 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             expect(defaultMessageElResubmit).to.be.ok();
           });
         });
+
         it('should show allow toggling columns from the expanded document', async function () {
           await PageObjects.discover.clickNewSearchButton();
-          await testSubjects.click('dscExplorerCalloutClose');
           await retry.try(async function () {
+            log.debug('row index: ' + String(rowToInspect - 1));
```

Comment:
guess we can remove the debugging, unless it might be useful for future failures?